spezi-invitation-code/index.js

```diff
@@ -0,0 +1,72 @@
+//
+// This source file is part of the StudyApplication based on the Stanford Spezi Template Application project
```

Comment:
We should adjust this to reflect the headers in this repo. Applies to all other headers as well.